witness/src/Dscp/Snowdrop/Actions.hs

```diff
@@ -37,6 +40,10 @@ deriving instance MonadUnliftIO (SD.RIO ctx)
 type SDDataActions  m = SD.DbModifyActions (AVLChgAccum AvlHash Ids Values) Ids Values m AvlProof
 type SDBlockActions m = SD.DbModifyActions (SD.SumChangeSet Ids Values) Ids Values m ()
 
+data AVLPlusBlockComposition
```

Comment:
How does it differ from `BlockPlusAVLComposition` imported from `Dscp.Snowdrop.Configuration`, except the order of the... arguments to `SD.CompositeChgAccum` in `sdActionsComposition`?
What's going on, generally?

Reply:
I actually moved `BlockPlusAVLComposition` here and renamed it.
`Composition A B` isn't symmetric - we have to provide the set of prefixes `A` needs access to, all other prefixes are assumed to relate to `B`. The overhead of the composition scales with the logarithm of the size of that set. To decrease its size I swapped `A` and `B`, thus `BlockPlusAVLComposition` should've been also renamed.